commons/addons/mongodb.md

```diff
@@ -21,6 +21,33 @@ The version currently installed by the add-on is :
 - on shared plans (Peanut) : MongoDB 3.2.9
 - on newly created dedicated databases (plans Hazelnut and above) : MongoDB 3.6.4
 
+## About Free Dababases
+
+Free plans are meant for tests and development usage only. Using these databases in production is not recomanded, because actual performances may vary depending on the global cluster's usage. Also, before switching to production, consider to upgrade to a dedicated database for better performances.
+
+### Important Note About Fair Use on Free Databases
+
+Heavy usage of free databases may impact the shared cluster they rely upon, degrading performance of other shared databases. To that extent, we reserve ourselves the right to disconnect databases making a high usage, **above 15 operations/seconds**, if no answer is received from you after a first notice.
```

Comment:
Please keep sentences under 17 words.

```
Heavy usage of free databases may impact the shared cluster they rely upon. It will degrade performances of the other databases. To that extent, *above 15 operations/second*, we reserve ourselves the right to disconnect your databases. This will however happen after making contact with your teams.
```